tox.ini

```
@@ -1,14 +1,25 @@
 [tox]
-envlist = py{36,37,38,39}
+envlist = py{36,37,38,39},pypy3,lint
```

```suggestion
envlist = py{36,37,38,39,py3},lint
```
I think you can do this